torch/csrc/jit/fuser/executor.cpp

```diff
@@ -323,7 +335,11 @@ bool runFusion(const int64_t key, Stack& stack) {
   // Tries to run fallback if map size can't be computed
   if (!maybe_map_size)
     return false;
-  expandArgs(spec, inputs, *maybe_map_size);
+  if (spec.hasRandom()) {
+      bool hasBroadcast = expandArgs(spec,inputs, *maybe_map_size, false);
```

Comment:
The naming for this is confusing, it took me a few minutes to figure out what was going on. Could you rename this to `shouldExpandArgs` (and also keep the `expandArgs` function?)